apps/admin-gui/src/app/vos/pages/member-detail-page/member-attributes/member-attributes.component.html

```diff
@@ -63,7 +27,25 @@ <h1 class="page-subtitle">{{'MEMBER_DETAIL.SETTINGS.ATTRIBUTES.TITLE' | translat
       </ng-template>
       <ng-template matTabContent>
         <app-two-entity-attribute-page [firstEntity]="'member'" [firstEntityId]="memberId"
-                                       [secondEntity]="'resource'" *ngIf="!loading"></app-two-entity-attribute-page>
+                                       [secondEntity]="'resource'"></app-two-entity-attribute-page>
+      </ng-template>
+    </mat-tab>
+    <mat-tab class="mb-2">
+      <ng-template matTabLabel>
+        {{'ADMIN_USER.ENTITY' | translate}}
+      </ng-template>
+      <app-one-entity-attribute-page [entityId]="this.member.userId" [entity]="'user'">
+      </app-one-entity-attribute-page>
```

Comment:
Wrap the body with \<ng-tamplate matTabContent\> tag so it is lazily loaded. Otherwise the member can be undefined when accessing userId.